modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/NodeImpl.java

```diff
@@ -571,9 +570,32 @@ public NodeImpl(
         this.confCtx = new ConfigurationCtx(this);
         this.wakingCandidate = null;
         this.applyCommittedFuture = new CompletableFuture<>();
-        this.ownFsmCallerExecutorDisruptorConfig = ownFsmCallerExecutorDisruptorConfig;
+        this.ownFsmCallerExecutorDisruptorConfig = null;
     }
 
+        public NodeImpl(
```

Comment:
Same thing: let's reuse one constructor in the other. This seems to be especially important in this class, as the constructor has some non-trivial logic, not just assignments to fields, and it seems scary to duplicate this logic.

Reply:
Fix it